app/src/main/java/com/github/swent/echo/data/repository/datasources/Cache.kt

```diff
@@ -0,0 +1,21 @@
+package com.github.swent.echo.data.repository.datasources
+
+import android.app.Application
+import androidx.room.Room
+import com.github.swent.echo.data.room.AppDatabase
+import com.github.swent.echo.data.room.Converters
+
+class Cache(application: Application) : LocalDataSource {
```

Comment:
I'd suggest to rename this to `RoomDataSource` and move it to the room/ dicrectory/package for it to be consistent :)